src/validation/validateSubmissions.ts

```diff
@@ -106,70 +106,72 @@ function validateSubmissionSearchBySpokenLanguage(
     }
 }
 
-function validateSearchByName(
+function validateSubmissionName(
     filters: gqlTypes.SubmissionSearchFilters,
-    validateSearchResults: Result<unknown>
+    validateNameResults: Result<unknown>
 ): void {
     const name = filters.healthcareProfessionalName
 
-    if (!name) {
+    if (name === undefined) {
```

Comment:
I figured out why you did this and it makes sense, but wasn't intuitive at first. Let's add a comment here to explain why undefined here, then null later

Reply:
@ermish I added some comments. Let me know what you think.

Reply:
looks good 👍